

# Message ID
uint16 msg_id

# Message Length
uint16 msg_len

# Sequence
uint16 seq

# Idle Time
uint8 idle_t

# Time Status
GpsTimeStat time_stat

# GPS Week Number and Milliseconds from the beginning of the GPS week.
uint16 gps_week
uint16 gps_ms

# Receiver Status
uint32 rcv_stat_n
ReceiverStat rcv_stat

# Receiver Software Build Number
uint32 rcv_sw_v
